protostar/cheatable_starknet/controllers/transaction_info.py

```diff
@@ -0,0 +1,115 @@
+from dataclasses import dataclass
```

Comment:
Test which would be beneficial here
- deploys contracts A and B
- spoofs contract B
- calls contract A, A calls B b checks if its spoofed

Reply:
Added